packer/github-actions-runner/sources.pkr.hcl

```diff
@@ -9,7 +9,7 @@ source "amazon-ebs" "github-actions-runner" {
   iam_instance_profile                      = "bcda-mgmt-github-actions"
 
   source_ami_filter {
-    filters = { name = "${var.ami_filter}" }
+     filters = { name = "${var.ami_filter}" }
```

Comment:
The check in the PR is still building with the AL2 image instead of AL2023 because /github-runner/gold-image-filter in Parameter Store is set to "amzn2legacy-gi-*". As noted in my earlier comments, I'd rather hardcode the filter string (now "al2023-legacy-gi-*") here and avoid passing it in as a variable.